### ISSUE TITLE
Regarding to the consistency of Panoptic Segmentation annotation

### ISSUE THREAD

**Issue by ndinhtuan** [mcahny/vps#26]
Dear author of Cityscapes-VPS, 
thank you very much for your effort to implement the idea on Video Panoptic Segmentation, especially with the new dataset, Cityscapes-VPS. I am currently researching related to this field, which is favored a lot by your dataset. However, after running your scripts to divide and prepare dataset, I do analysis on the dataset, I have see something noticeable in the ground truth of dataset. Some instance masks of the car class is missing:

With this image, I take one example from validation set, you can see that some masks of car class in the right hand side is missing, which is the dark color. The image I took is from val/panoptic_video, which will be used for evaluation. (The image's name is 0125_0745_frankfurt_000001_029221_final_mask.png)
![Image](https://github.com/user-attachments/assets/a0b01246-e003-4294-b4c6-b0152fe953e5)

This leads me wandering whether it also happens in the training set. I have a look at train/inst, which will be used to produce labelmap for training. The image below also illustrates that some masks of the car class are missing on the right hand side. (The image's name is 0029_0174_frankfurt_000000_012131_original_image_instance_greyscale.png)

![Image](https://github.com/user-attachments/assets/f314a6a5-3bf1-4570-a3d1-4420368570ba)

Do you have any idea why it happens ?
Thank you very much for your effort again.
